packages/material-ui/tsconfig.build.json

```diff
@@ -0,0 +1,15 @@
+{
+  // This config is for emitting declarations (.d.ts) only
+  // Actual .ts source files are transpiled via babel
+  "extends": "./tsconfig",
+  "compilerOptions": {
+    "declaration": true,
+    "noEmit": false,
+    "emitDeclarationOnly": true,
+    "outDir": "build",
+    "rootDir": "./src",
+    "types": ["react"]
```

Comment:
This types section is not needed if we have installed ‘@types’ packages

Reply:
We have a bunch of types installed that change the global namespace. E.g. `@types/react-native`. We need to whitelist the types we need in order to avoid duplicate type mismatches.